package.json

```diff
@@ -41,9 +41,12 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm test"
+      "pre-commit": "lint-staged"
     }
   },
+  "lint-staged": {
+    "*.{js,jsx,md}": ["npm test"]
```

Comment:
We should separate this config, we already have a lint command for each file type.
It doesn't make sense to lint JS if you only updated docs.

Reply:
Sounds wise, will update today